src/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.csproj

```diff
@@ -5,6 +5,10 @@
     <IncludeInPackage>Microsoft.ML.TimeSeries</IncludeInPackage>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML.Mkl.Redist" Version="1.0.0" />
```

Comment:
how was timeseries working so far without adding this dependency?

Reply:
It works with the nuget, not in samples repo for some time series samples that require mkl. I have fixed it now.,